dracut.usage.asc

```diff
@@ -151,7 +151,7 @@ If your root partition is on the network see <<NetworkBoot>>.
 If you have to input passwords for encrypted disk volumes, you might want to set
 the keyboard layout and specify a display font.
 
-A typical german kernel command would contain:
+A typical German kernel command would contain:
```

Comment:
isn't this an adjective and therefore lowercase? .. a real fix would be: s/kernel command/kernel command line/

Reply:
ah, thanks, that's indeed a mistake on my side (and I completely overlooked the missing 'line')